<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWin</class>
 <widget class="QDialog" name="GraphWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lb_aiport_name_info">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_year">
      <attribute name="title">
       <string>Загруженность за год</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="graphYear" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_month">
      <attribute name="title">
       <string>Загруженность за месяц</string>
      </attribute>
      <widget class="QCustomPlot" name="graphMonth" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>511</width>
         <height>511</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cB_months">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>20</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Январь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Февраль</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Март</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Апрель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Май</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Июнь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Июль</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Август</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сентябрь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Октябрь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ноябрь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Декабрь</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>618</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>C:\Users\Ice7\Documents\QtRes\InspektorAeroportov\qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
